src/scss/application.scss

```diff
@@ -16,6 +18,30 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+* {
+  font-family: Roboto;
+  color: $black;
+}
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
+
+input {
```

Comment:
this is not right, not all inputs commonly have this style, this will be taken for every input on the app, there is date inputs, number inputs, etc.